### ISSUE TITLE
Helm chart for elasticsearch-exporter

### ISSUE THREAD

**Issue by svenmueller** [justwatchcom/elasticsearch_exporter#119]
Hi,

FYI, i've created a PR for the k8s helm chart repo to add a Helm chart for elasticsearch_exporter. (sadly reviewing takes way longer than i expected). Feel free to mention it in the installation part of the README when it's considered as stable (needs approval).

https://github.com/kubernetes/charts/pull/2525

-- Sven

**Comment by dominikschulz** [comment]
Thank you very much! We'll probably add this to the README once the PR is merged.

**Comment by svenmueller** [comment]
FYI: the helm chart was merged into the stable repo: https://github.com/kubernetes/charts/tree/master/stable/elasticsearch-exporter

**Comment by metalmatze** [comment]
Nice. Thanks for the effort! 😊

**Comment by zwopir** [comment]
added to the documentation, thanks again
